### DISCUSSION TITLE
How is the form stepper constructed using the schema notation?

### DISCUSSION THREAD

**Discussion by deanCompRSA** [vueform/vueform#298] [Questions] [answered]
Looking at https://vueform.com/reference/form-steps

Instead of using the FormElements within the html itself. How would I do it using a schema object?

**Comment by Foxxat** [comment, marked as answer]
I only found this project myself a couple of days ago, so sorry in advance if this doesn't answer your question - but you can create a multistep form in the builder https://builder.vueform.com/demo.
If you then go-to the export panel, you change the output to schema - would this help ?

**Reply by deanCompRSA** [reply]
That answered. thanks mate.

For clarification to other reading, it looks like this: https://stackblitz.com/edit/github-rakass?file=src%2FApp.vue
